packages/fxa-auth-server/fxa-oauth-server/lib/keys.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
This is going to be a merge conflict nightmare with https://github.com/mozilla/fxa/pull/1668, but I volunteer to take on the rebase of that PR once this one has landed. Just a heads-up.